src/main/scala/zio/kafka/consumer/internal/ConsumerAccess.scala

```diff
@@ -31,21 +28,21 @@ private[consumer] class ConsumerAccess(
       }
       .fork
       .flatMap(fib => fib.join.onInterrupt(ZIO.effectTotal(consumer.wakeup()) *> fib.interrupt))
+
 }
 
 private[consumer] object ConsumerAccess {
-  type ByteArrayKafkaConsumer = KafkaConsumer[Array[Byte], Array[Byte]]
+  type ByteArrayKafkaConsumer = JConsumer[Array[Byte], Array[Byte]]
 
-  def make(settings: ConsumerSettings): RManaged[Blocking, ConsumerAccess] =
```

Comment:
:red_circle: 
So there isn't the old `make` anymore?

Reply:
It's for internal use only